testproto2/proto2.letmegrpc.go

```diff
@@ -472,7 +473,7 @@ s += '<label class="heading">Song</label>'
 s += '</div></div>'
 s += '<div class="field form-group"><label class="col-sm-2 control-label">Name: </label><div class="col-sm-10"><input class="form-control" name="Name" type="text" '+setStrValue("Type in a Name", json["Name"])+'/></div></div>';
 				
-s += '<div class="field form-group"><label class="col-sm-2 control-label">Track: </label><div class="col-sm-10"><input class="form-control" name="Track" type="number" step="1" '+setValue(1, json["Track"])+'/></div></div>';
+s += '<div class="field form-group"><label class="col-sm-2 control-label">Track: </label><div class="col-sm-10"><input class="form-control" name="Track" type="text" '+setStrValue("1", json["Track"])+'/></div></div>';
```

Comment:
But new int64s will always be written as a quoted string in the url, right?

Reply:
yep